R/tostring.R

```diff
@@ -1008,6 +1452,7 @@ propose_column_widths <- function(x, indent_size = 2) {
             # need to update widths to fit content with colspans
             # spread width among columns
             widths[i_cols] <- cw_i + spread_integer(nchar_i - available_width, length(cw_i))
+            ## widths[i_cols] <- cw_i + spread_width(nchar_i - available_width, length(cw_i))
```

Comment:
```suggestion
```
Did you mean to comment here?

Reply:
Because if commented this function (spread_width) does not seem used

Reply:
You're correct, spread_width spreads the remaining with allowing fractional allocation (ie if there are 3 spaces left and there are 5 columns including row labels, each column gets an extra 3/5 of a space width). 

I was initially intending to allow non-integer column widths, but that feature didn't get into this round of development. Still a nice-to-have in truetype space, as spaces are small but not *that* small, but so far integer colwidths seems to work ok because of how we are exporting rtf (we export an rtf table object, not blocks of text like `export_as_pdf` currently does).

Reply:
understood! So we keep it for now?